src/Components/Results/Programs/ProgramCard.tsx

```diff
@@ -34,6 +34,20 @@ const ProgramCard = ({ program }: ProgramCardProps) => {
 
   const isMobile = size < 769 ? true : false;
 
+  const newProgramFlagRef = useRef<HTMLDivElement | null>(null);
+  const [leftOffset, setLeftOffset] = useState('0');
+  const [newProgramFlagWidth, setNewProgramFlagWidth] = useState<number>(0);
+
+  useEffect(() => {
+    if (program.new && newProgramFlagRef.current) {
+      const width = newProgramFlagRef.current.offsetWidth;
+      const remWidth = width / 16;
```

Comment:
Can we add font size as a number to the style controller, and then divide by that. This breaks if we change the root font size.

Reply:
Also, `theme.fontSize` will need to be added to the `useEffect` dependencies.

Reply:
Thanks for pointing this out. After looking at the style controller realized the `font-size` is already part of `css-variables`. So I just use that value instead of creating a new one.